source/extensions/filters/common/expr/custom_cel/extended_request/utility/utility.cc

```diff
@@ -0,0 +1,80 @@
+#include "source/extensions/filters/common/expr/custom_cel/extended_request/utility/utility.h"
+
+#include "source/common/protobuf/protobuf.h"
+
+#include "eval/public/cel_value.h"
+#include "eval/public/containers/container_backed_list_impl.h"
+#include "eval/public/containers/container_backed_map_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Filters {
+namespace Common {
+namespace Expr {
+namespace CustomCel {
+namespace ExtendedRequest {
+namespace Utility {
+
+using google::api::expr::runtime::CelList;
+using google::api::expr::runtime::CelMap;
+using google::api::expr::runtime::CelValue;
+using google::api::expr::runtime::ContainerBackedListImpl;
+
+template <typename M> CelValue createCelMapImpl(Protobuf::Arena& arena, M& map);
+
+CelList* mergeLists(Protobuf::Arena& arena, const CelList* list1, const CelList* list2) {
```

Comment:
@kyessenov do we need to be doing stuff like this by hand?

Reply:
I didn't find a mergeLists (no duplicates).